chainer/training/trainer.py

```diff
@@ -165,6 +166,10 @@ def __init__(self, updater, stop_trigger=None, out='result',
         for ext in extensions:
             self.extend(ext)
 
+    @property
+    def is_before_training(self):
```

Comment:
Could you write a short documentation what this returns and that it can be used by extensions (for e.g. those who'd be interested in writing their own)?